staging/src/k8s.io/kubectl/pkg/util/templates/templater.go

```diff
@@ -90,7 +91,7 @@ func (templater *templater) HelpFunc() func(*cobra.Command, []string) {
 		t := template.New("help")
 		t.Funcs(templater.templateFuncs())
 		template.Must(t.Parse(templater.HelpTemplate))
-		out := term.NewResponsiveWriter(c.OutOrStdout())
+		out := c.OutOrStderr()
```

Comment:
Have to remove this and wrap each template string separately, or else for the help command `-o output`, `ResponsiveWriter` will move it to next line (as shown in the picture)

![image](https://user-images.githubusercontent.com/42656921/131731435-efde08f1-e600-4217-bde4-444290e1cb39.png)

Reply:
Discussed with @eddiezane , we agreed it's fine